src/platform/forms/save-in-progress/SaveInProgressIntro.jsx

```diff
@@ -45,9 +45,7 @@ class SaveInProgressIntro extends React.Component {
         const isExpired = expiresAt.isBefore();
 
         if (!isExpired) {
-          const lastSavedDateTime = moment
-            .unix(savedAt)
-            .format('M/D/YYYY [at] h:mm a');
+          const lastSavedDateTime = savedAt.format('M/D/YYYY [at] h:mm a');
```

Comment:
`savedAt` was turned into a moment type on line 40 so it's not necessary to convert it to `unix` type before formatting it.

Reply:
based on the history of this file, it looks like this has been a problem for at least four months? 😱